app.js

```diff
@@ -113,6 +113,11 @@ app.get('/runs', isUser, runs.list);
 app.get('/runs/:id', isUser, runs.show);
 
 app.post('/experimenters', experimenters.create);
+// Route to be released for experimenter update
+// app.put('/experimenters/:id', isUser, experimenters.update);
+
+// Route to use for testing
+app.put('/experimenters/:id', experimenters.update);
```

Comment:
OK--but let's switch to the route with middleware as soon as possible. We should make a check on :id being equal to the session user id--see below.